resources/faq.md

```diff
@@ -0,0 +1,163 @@
+# FAQ
+
+This FAQ is still a work in progress.<br>
+Do not hesitate to ask us your own questions. It will help us improve this page!
+
+## I have never used a search engine before. Can I use MeiliSearch anyway?
+
+Of course! No need to get used with ElasticSearch or Solr to handle MeiliSearch.
+
+MeiliSearch is really **easy to use** and is accessible for all kinds of developers!
+
+[A quick tour](/guides/introduction/quick_start_guide.md) is available in this documentation to get the basics of MeiliSearch.
```

Comment:
[Take a quick tour](/guides/introduction/quick_start_guide.md) to learn the basics of MeiliSearch.